edisgo/network/timeseries.py

```diff
@@ -2186,6 +2186,19 @@ def resample_timeseries(
         # set new timeindex
         self._timeindex = index
 
+    def scale_timeseries(
```

Comment:
Why are you not using the scaling functionality in EDisGo.analyze? I find it a bit dangerous to change the timeseries in the TimeSeries object because it could lead to loosing the original time series data.